app/decorators/miq_policy_decorator.rb

```diff
@@ -18,7 +18,7 @@ def fonticon
            when 'ExtManagementSystem'
              'pficon pficon-server'
            when 'PhysicalServer'
-             'pficon pficon-enterprise'
+             PhysicalServerDecorator.fonticon
```

Comment:
Use`.safe_constantize.try(:decorate).try(:fonticon)` instead and then you can drop the `case`.